yt/python/yt/wrapper/schema/types.py

```diff
@@ -109,6 +112,61 @@ def create_annotated_type(py_type, ti_type, to_yt_type=None, from_yt_type=None):
     return Annotated[py_type, Annotation(ti_type, to_yt_type=to_yt_type, from_yt_type=from_yt_type)]
 
 
+def create_yt_enum(py_type, ti_type, to_yt_type=None, from_yt_type=None):
```

Comment:
Type annotations will make work easier (it not so easy to understand what `ti_type` is)